{"name": "test_popular[u]", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: './data/login.yaml'", "trace": "@pytest.fixture()\n    def login():\n        path=\"/api/user/snackUser/login\"\n        url1=protocol+host+path\n>       data=yaml_conf.yaml_load(\"./data/login.yaml\")\n\ntest_popular.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nyaml_path = './data/login.yaml'\n\n    def yaml_load(yaml_path):\n>       with open(yaml_path, \"r\", encoding=\"utf-8\") as f:\nE       FileNotFoundError: [Errno 2] No such file or directory: './data/login.yaml'\n\n..\\common\\yaml_conf.py:9: FileNotFoundError"}, "parameters": [{"name": "pam1", "value": "'u'"}], "start": 1639489210901, "stop": 1639489210901, "uuid": "552adfd7-17f5-4f2b-82e6-c7920e8c1f4b", "historyId": "85dc3aa114c79abc247d5b3c3a4bded6", "testCaseId": "e782efbbe0a08f019148996912e5ca9a", "fullName": "testcase.test_popular#test_popular", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_popular"}, {"name": "host", "value": "CYHD-L-0025"}, {"name": "thread", "value": "17228-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_popular"}]}